Octokit/Models/Request/RequestParameters.cs

```diff
@@ -49,6 +49,9 @@ static List<PropertyParameter> GetPropertyParametersForType(Type type)
             Justification = "GitHub API depends on lower case strings")]
         static Func<PropertyInfo, object, string> GetValueFunc(Type propertyType)
         {
+            // get underlying type if nullable
+            propertyType = Nullable.GetUnderlyingType(propertyType) ?? propertyType;
```

Comment:
Without this line it does not go to `if (propertyType.IsEnumeration())` and somehow produces "All" instead of "all" + does not use `[Parameter]` attribute.